src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties

```diff
@@ -718,15 +726,18 @@ C2CAD1457536032C387B80EBEFEAE09D=GAE Studio vous aide à optimiser vos applicati
 CAB5613DE3E228D4FB7F6A8D2602632A=Supporte <span>plusieurs langages</span>
 
 # Description: Chose what you want to see
+# TODO: DEPRECATED (CONSIDER REMOVING)
 CDC8904A3656A43B67DD5C70EEAD6F35=Choisissez ce que vous voulez voir
 
 # Description: Using real-time analysis of datastore requests, GAE Studio...
 DEED6AA7544D8096C42D4C7EBEFD674B=Utilisant l''analyse en temps réel des requêtes des bases de données, GAE Studio cible rapidement celles qui sont inefficaces et fournit des indicateurs de performance aux développeurs afin qu''ils puissent réduire les temps de latence.
 
 # Description: Your metrics at work
+# TODO: DEPRECATED (CONSIDER REMOVING)
```

Comment:
yep, well the gae page in now visually the same as the other products pages, this was the text in the btn to open the "drawer" in the old visual, it should be rmv when I pull master